CHANGELOG.md

```diff
@@ -19,6 +19,10 @@ Improvement:
 * Add room_details api
 * Add room_members api
 
+Bug fixes:
```

Comment:
This is actually a breaking change.

Reply:
:+1: Moved to the breaking change section

Reply:
Could the entry be more explicit about why it's a breaking change? Like saying that it's a new type without `validated_at` and `added_at`.

Reply:
Changed. Is it clearer now?

Reply:
Yes, thanks!